includes/discovery/bgp-peers.inc.php

```diff
@@ -12,7 +12,7 @@
 }
 
 if (empty($bgpLocalAs)) {
-    $bgpLocalAs = snmp_getnext($device, 'bgpLocalAs', '-OQUsv', 'BGP4-MIB');
+    $bgpLocalAs = snmp_get($device, 'bgpLocalAs.0', '-OQUsv', 'BGP4-MIB');
```

Comment:
Bizarre, it goes back to the original code using a walk to fetch this value when it is just a scalar.  Probably test data will need to be corrected.

```suggestion
    $bgpLocalAs = \SnmpQuery::get('BGP4-MIB::bgpLocalAs.0')->value();
```